### ISSUE TITLE
Duplicated lines

### ISSUE THREAD

**Issue by jgarec** [felipebz/sonar-plsql#69]
Hi,

First of all, thx for this plugin.
We are currently testing this plugin for sonar integration and it sounds good.

I'm able to fill some of the metrics (bugs ; code smells ; coverage ; unit tests). Is this plugin able to detect duplicate code ? I read Unit tests and ITs to find examples and it seems that it only checks that duplication is zero.
`assertThat(getMeasureAsDouble(PROJECT_KEY, "duplicated_lines")).isZero();`

Do you have an example which gives a non zero metric for duplication ?

BR

**Comment by felipebz** [comment]
Hi @jgarec.

There's no IT checking this, but currently SonarQube does report duplications for PL/SQL code.

I just did this test on my machine:

```
git clone https://github.com/mortenbra/alexandria-plsql-utils.git
cd alexandria-plsql-utils
sonar-scanner -Dsonar.projectKey=alexandria -Dsonar.sources=.
```

And the result:

![image](https://user-images.githubusercontent.com/13829/39014546-095c5d78-43f1-11e8-9474-b5d3281040c4.png)

**Comment by jgarec** [comment]
Hi,

Found some duplicated code inside this project and add them inside my test project.
So everything is ok  :)

![image](https://user-images.githubusercontent.com/16031094/39248580-c5b02284-489c-11e8-96d9-853bff6d02c9.png)


Br,
Jérôme
